src/partials/how-we-do.html

```diff
@@ -0,0 +1,118 @@
+    <section class="how-we-do__section">
+      <div class="container">
+        <h1 class="how-we-do__heading">How do we do it</h1>
+        <ul class="how-we-do__list">
+          <li class="how-we-do__item">
+            <div class="how-we-do__svg-list">
```

Comment:
Każdy div "how-we-do__svg-list" jest niepotrzebny. Padding, który nadaje od góry należy przenieść do how-we-do__item. modyfikator --active z fill należy napisać dla how-we-do-svg.